File: SecondYear/serv-er_SarveshBaranwal_2226cseAI&ml1045_2/Week_1/forLoop.cpp
#include<stdio.h>
using namespace std;

int main()
{
        /*
        *  Write your code here. 
         *  Read input as specified in the question.
         *  Print output as specified in the question.
        */
        int a=0,b=1,d,c,i;
        scanf("%d",&c);
        if (c==1){
                d=1;
        }
        for(i=1;i<c;i++){
                d=b+a;
                a=b;
               b=d;

        } 
        printf("%d",d);
}

File: SecondYear/aayushraghav93_AayushRaghav_2226CSE(AI)1108_2/Week_1/For_loop.java


import java.util.*;
public class For_loop {
	static int fib(int n)
    {
        if (n==0)
            return 0;
      else if(n==1 || n==2)
            return 1;
        return fib(n - 1) + fib(n - 2);
    }


	public static void main(String[] args) {
		 Scanner sc=new Scanner(System.in);
		 int n=sc.nextInt();
        System.out.println(fib(n));
		
		/* Your class should be named Solution.
	 	* Read input as specified in the question.
	 	* Print output as specified in the question.
		*/

		
	}

}